frontend/tests/e2e/search/search.spec.ts

```diff
@@ -156,12 +156,12 @@ test.describe("Search page tests", () => {
     page,
   }: PageProps) => {
     await page.goto("/search");
+    await waitForSearchResultsInitialLoad(page);
```

Comment:
not sure why, but these tests got flaky. Adding the wait on initial page load seems to solve the problem